### PR TITLE
Correct permmap stating that sendto is a read.

### PR DESCRIPTION
**Pull request by karlwmacmillan**
sendto on some object classes was marked as a read when it should be
a write.

### CODE REVIEW

File: KNOWN-BUGS [hunk 1]
```diff
@@ -5,4 +5,4 @@ The following is a list of known bugs with SETools.
 
   'Type' object is not iterable
 
-  This message may be seen multiple times at the end of running unit tests.
+  This message may be seen multiple times at the end of running unit tests.
```
Comment (pebenito) [inline KNOWN-BUGS]: Please remove this unnecessary whitespace change.

**Comment by karlwmacmillan** [comment]
Sorry about that - not sure how that change slipped in.
